internal/convert/convert_test.go

```diff
@@ -1,458 +0,0 @@
-// Package convert is the package that contains the convert functionality.
-package convert
-
-import (
-	"testing"
-
-	"github.com/aiven/aiven-go-client/v2"
-	"github.com/google/go-cmp/cmp"
-
-	"github.com/aiven/go-api-schemas/internal/pkg/types"
-	"github.com/aiven/go-api-schemas/internal/pkg/util"
-)
-
-// TestUserConfigSchema tests the UserConfigSchema function.
-// nolint:funlen,lll // This function is long, but it's a test function.
-// // These lines are long, but they're test data.
-func TestUserConfigSchema(t *testing.T) {
-	type args struct {
-		v aiven.UserConfigSchema
-	}
-
-	tests := []struct {
-		name    string
-		args    args
-		want    *types.UserConfigSchema
-		wantErr error
-	}{
-		{
-			name: "basic",
-			args: args{
-				v: aiven.UserConfigSchema{
-					Title:                "",
-					Description:          "",
-					Type:                 nil,
-					Default:              nil,
-					Required:             nil,
-					Properties:           nil,
-					AdditionalProperties: nil,
-					Items:                nil,
-					OneOf:                nil,
-					Enum:                 nil,
-					Minimum:              util.Ref(0.0),
-					Maximum:              util.Ref(0.0),
-					MinLength:            util.Ref(0),
-					MaxLength:            util.Ref(0),
-					MaxItems:             util.Ref(0),
-					CreateOnly:           false,
-					Pattern:              "",
-					Example:              nil,
-					UserError:            "",
-				},
-			},
-			want: &types.UserConfigSchema{
-				UserConfigSchemaDeprecationInfo: types.UserConfigSchemaDeprecationInfo{},
-				Title:                           "",
-				Description:                     "",
-				Type:                            nil,
-				Default:                         nil,
-				Required:                        nil,
-				Properties:                      nil,
-				Items:                           nil,
-				OneOf:                           nil,
-				Enum:                            []types.UserConfigSchemaEnumValue{},
-				Minimum:                         util.Ref(0.0),
-				Maximum:                         util.Ref(0.0),
-				MinLength:                       util.Ref(0),
-				MaxLength:                       util.Ref(0),
-				MaxItems:                        util.Ref(0),
-				CreateOnly:                      false,
-				Pattern:                         "",
-				Example:                         nil,
-				UserError:                       "",
-			},
-			wantErr: nil,
-		},
```

Comment:
Would be nice to have some tests for the updated code

Reply:
Agree, but they are barely maintainable and readable. I think the generation + git diff do a better job.